deploy/role.yaml

```diff
@@ -3,47 +3,106 @@ kind: ClusterRole
 metadata:
   name: unifiedpush-operator
 rules:
+
+# The resources in this group are the CRs that the controllers in this
+# operator react to. Nothing in the operator is expected to create or
+# delete CRs, so those permissions are not given.
+- apiGroups:
+  - push.aerogear.org
+  resources:
+  - '*'
+  verbs:
```

Comment:
are those all the verbs? if so can't you use  `*`?

Reply:
nvm, patch is not in the list :)

Reply:
Also delete is not there. Actually 'create' should maybe not be there either (see comment above this section, I intended to remove it)